genesis/process/genesisBlockCreator.go

```diff
@@ -446,19 +449,19 @@ func (gbc *genesisBlockCreator) createHeaders(args *headerCreatorArgs) (map[uint
 	return genesisBlocks, nil
 }
 
-func (gbc *genesisBlockCreator) computeInitialDNSAddresses(enableEpochsConfig config.EnableEpochs) error {
+func (gbc *genesisBlockCreator) computeInitialDNSAddresses(epochsConfig config.EpochConfig) error {
```

Comment:
Not ok that we have to change all mainnet usage of these configs just for sovereign.
Mainnet only needs `EpochConfigs`.
This is related to my comments from above

Reply:
Reverted